db/map_index.txt

```diff
@@ -642,8 +642,8 @@ ve_fild04
 ve_fild05
 ve_fild06
 ve_fild07
-poring_c01
-poring_c02
+//poring_c01
+//poring_c02
 que_ng
```

Comment:
In order to avoid issues with saved data, indexes of existing maps shouldn't ever change - can you force `que_ng` to keep its old index? (I believe it should be `que_ng 630`, but please double-check on your side)